framework/src/controller/schema/constants.js

```diff
@@ -3,177 +3,31 @@ module.exports = {
 		id: 'constants',
 		type: 'object',
 		required: [
-			'ACTIVE_DELEGATES',
-			'BLOCK_SLOT_WINDOW',
-			'ADDITIONAL_DATA',
-			'BLOCK_RECEIPT_TIMEOUT',
 			'EPOCH_TIME',
-			'FEES',
-			'MAX_PAYLOAD_LENGTH',
-			'MAX_PEERS',
-			'MAX_SHARED_TRANSACTIONS',
+			'BLOCK_TIME',
 			'MAX_TRANSACTIONS_PER_BLOCK',
-			'MAX_VOTES_PER_TRANSACTION',
-			'MAX_VOTES_PER_ACCOUNT',
-			'MIN_BROADHASH_CONSENSUS',
-			'MULTISIG_CONSTRAINTS',
-			'NETHASHES',
-			'NORMALIZER',
 			'REWARDS',
-			'TOTAL_AMOUNT',
-			'TRANSACTION_TYPES',
-			'UNCONFIRMED_TRANSACTION_TIMEOUT',
-			'EXPIRY_INTERVAL',
 		],
 		properties: {
-			ACTIVE_DELEGATES: {
-				type: 'number',
-				format: 'oddInteger',
-				min: 1,
-				default: 101,
-				description: 'The default number of delegates allowed to forge a block',
-			},
-			BLOCK_SLOT_WINDOW: {
-				type: 'integer',
-				min: 1,
-				default: 5,
-				description: 'The default number of previous blocks to keep in memory',
-			},
-			ADDITIONAL_DATA: {
-				type: 'object',
-				required: ['MIN_LENGTH', 'MAX_LENGTH'],
-				properties: {
-					MIN_LENGTH: {
-						type: 'integer',
-						min: 1,
-						default: 1,
-						description: 'Additional data (Min length)',
-					},
-					MAX_LENGTH: {
-						type: 'integer',
-						min: 1,
-						default: 64,
-						description: 'Additional data (Max length)',
-					},
-				},
-				default: {
-					MIN_LENGTH: 1,
-					MAX_LENGTH: 64,
-				},
-			},
-			BLOCK_RECEIPT_TIMEOUT: {
-				type: 'integer',
-				min: 1,
-				default: 20, // 2 blocks
-				description: 'Seconds to check if the block is fresh or not',
-			},
 			EPOCH_TIME: {
 				type: 'string',
 				format: 'date-time',
 				default: new Date(Date.UTC(2016, 4, 24, 17, 0, 0, 0)).toISOString(),
 				description:
 					'Timestamp indicating the start of Lisk Core (`Date.toISOString()`)',
 			},
-			FEES: {
-				$ref: 'fees',
-				default: {
-					SEND: '10000000',
-					VOTE: '100000000',
-					SECOND_SIGNATURE: '500000000',
-					DELEGATE: '2500000000',
-					MULTISIGNATURE: '500000000',
-					DAPP_REGISTRATION: '2500000000',
-					DAPP_WITHDRAWAL: '10000000',
-					DAPP_DEPOSIT: '10000000',
-				},
-			},
-			MAX_PAYLOAD_LENGTH: {
-				type: 'integer',
-				min: 1,
-				default: 1024 * 1024,
-				description:
-					'Maximum transaction bytes length for 25 transactions in a single block',
-			},
```

Comment:
Are we throwing away the validation rules we wrote here?

Reply:
We don't need to validate as those values won't be changed by the end user. We will slowly move those values back to the original file when we remove hardcodings in the codebase